examples/image-streams/image-streams-centos7.json

```diff
@@ -22,6 +22,7 @@
               "description": "Build and run Ruby 2.0 applications",
               "iconClass": "icon-ruby",
               "tags": "builder,ruby",
+              "supports": "ruby:2.0,ruby",
```

Comment:
I don't get the difference between "we'll first check for IS ruby and tag 2.0, then search IS ruby for tag 2.0" isn't that the same? But yes, I'm using a weighted resolver to find the right image: https://github.com/openshift/origin/pull/3057/files#diff-67f42376bdba60c035c6268d160059baR268